### PR TITLE
fix(docker): raise events instead of exceptions while destroying containers

### PR DESCRIPTION
**Pull request by enaydanov**
Trello: https://trello.com/c/wnDeD5IF/1878-autossh-containers-are-left-after-the-test-finished-suceessfully

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [x] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/utils/docker.py [hunk 1]
```diff
@@ -216,7 +226,14 @@ def destroy_container(cls, instance: object, name: str, ignore_keepalive: bool =
     @classmethod
     def destroy_all_containers(cls, instance: object, ignore_keepalive: bool = False) -> None:
         for name in tuple(instance._containers.keys()):
-            cls.destroy_container(instance, name, ignore_keepalive=ignore_keepalive)
+            try:
+                cls.destroy_container(instance, name, ignore_keepalive=ignore_keepalive)
+            except Exception:  # pylint: disable=broad-except
```
Comment (fruch) [inline sdcm/utils/docker.py:231]: please notice, that we don't track those event on other steps like clear-resources and such.

Comment (enaydanov) [reply sdcm/utils/docker.py]: Ok, I've made it little bit noisier.

Comment (bentsi) [reply sdcm/utils/docker.py]: @fruch where will those events be saved? will they be added to `critical.log`?

Comment (fruch) [reply sdcm/utils/docker.py]: Looks like no where...

**Pull request merged**